src/forms/TimeInput.jsx

```diff
@@ -2,6 +2,40 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import cx from 'classnames';
 
+import Flex from '../layout/Flex';
+import FlexItem from '../layout/FlexItem';
+import SelectInput from './SelectInput';
+
+export const HOURS_INPUT_CLASS = 'timeInput-hours';
+export const MINUTES_INPUT_CLASS = 'timeInput-minutes';
+export const MERIDIAN_INPUT_CLASS = 'timeInput-meridian';
+
+export const HOURS = 'hours';
+export const MINUTES = 'minutes';
+export const MERIDIAN = 'meridian';
+
+const formatDigits = (number) => `0${number}`.slice(-2);
+
+/**
+ * @description takes a time value and returns individual parts
+ * @param {String} time - time value in the format 'hh:mm'
+ * @param {String} part - a string corersponding to a part, optional
+ * @returns {String|Object} the only parsed value or all the parsed parts if no part defined
+ */
+export const getTimeParts = (time, part) => {
+	const [ hours, minutes ] = time.split(':');
+	const parts = {
+		[HOURS]: hours,
+		[MINUTES]: minutes,
+		[MERIDIAN]: hours < 12 ? 'AM': 'PM',
+	};
+	return (part) ? parts[part] : parts;
+};
+const formatHours = (hours, meridian) => {
+	const newHours = meridian ? ((hours % 12) + (12 * (meridian === 'PM'))) : hours % 24 ;
```

Comment:
Looks like you might not need to use `(hours % 12)` - just `hours`, unless you really want to support `formatHours(15, 'AM') === '03'`.